test/fixtures/compile-test-data.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Exposing it in package.json could be good. Or a couple comment lines within this file. Or markdown documentation in this directory.